openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/Openc3Screen.vue

```diff
@@ -178,14 +178,16 @@ import upperFirst from 'lodash/upperFirst'
 import camelCase from 'lodash/camelCase'
 import EditScreenDialog from './EditScreenDialog'
 
+const MAX_ERRORS = 20
```

Comment:
I like this ... and it could probably even be set lower. I think a future enhancement would be to filter out cascading errors somehow.